src/blocks/stars.scss

```diff
@@ -0,0 +1,39 @@
+.card__rating {
+  width: 166px;
+  height: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &--margin--top {
+    margin-top: 16px;
+  }
+}
+
+.card__stars-area {
+  padding: 0;
+  display: flex;
+  gap: 4px;
+  list-style: none;
+}
+
+.card__stars {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(../images/star.svg);
+  background-size: auto;
+
+  &:nth-child(-n + 4) {
```

Comment:
The `:nth-child(-n + 4)` selector will apply the active star image to the first four stars. Ensure this matches the intended logic for displaying active stars based on the rating value.